### PR TITLE
release: 2.2.0

### PR DESCRIPTION
**Pull request by bastimeyer**
Let's prepare the `2.2.0` release.

Anything missing that needs to get added? The PR list is pretty much empty now.

Please check the changelog for any mistakes or missing changes which are important.
https://deploy-preview-3800--streamlink.netlify.app/changelog.html

### CODE REVIEW

**Review by back-to — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,87 @@
...
+
+Release highlights:
+
+- Changed: default config file path on macOS and Windows ([#3766](https://github.com/streamlink/streamlink/pull/3766))  
+  - macOS: `${HOME}/Library/Application Support/streamlink/config`
+  - Windows: `%APPDATA%\streamlink\config`
+- Changed: default custom plugin directory path on macOS and Linux/BSD ([#3766](https://github.com/streamlink/streamlink/pull/3766))
+  - macOS: `${HOME}/Library/Application Support/streamlink/plugins`
+  - Linux/BSD: `${XDG_DATA_HOME:-${HOME}/.local/share}/streamlink/plugins`
+- Deprecated: old config file paths and custom plugin directory paths ([#3784](https://github.com/streamlink/streamlink/pull/3784))
```
Comment (gravyboat) [inline CHANGELOG.md:13]: @bastimeyer I think the changelog should be more clear that we are in the process of deprecating the old paths and they will be deprecated on the next major release. This makes it seem like we're dropping support for them today.

Comment (bastimeyer) [reply CHANGELOG.md]: https://github.com/streamlink/streamlink/compare/fd00fc42b7b26f01f4176821f577d84661e5e65e..dac69bbb70e5285a7a3183c25fd395292d53b051

Comment (bastimeyer) [reply CHANGELOG.md]: If you want, we can explicitly list the deprecated paths, but as said, I'm not sure if it's too much. In the current version, it doesn't mention `~/.streamlinkrc` for example.

Comment (bastimeyer) [reply CHANGELOG.md]: What do you think about this? Is it too much?

```diff
diff --git a/CHANGELOG.md b/CHANGELOG.md
index 7c7ef2ae..fc8b7a8f 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -11,6 +11,16 @@ Release highlights:
   - macOS: `${HOME}/Library/Application Support/streamlink/plugins`
   - Linux/BSD: `${XDG_DATA_HOME:-${HOME}/.local/share}/streamlink/plugins`
 - Deprecated: old config file paths and old custom plugin directory paths ([#3784](https://github.com/streamlink/streamlink/pull/3784))  
+  - Windows:
+    - `%APPDATA%\streamlink\streamlinkrc`
+  - macOS:
+    - `${XDG_CONFIG_HOME:-${HOME}/.config}/streamlink/config`
+    - `${XDG_CONFIG_HOME:-${HOME}/.config}/streamlink/plugins`
+    - `${HOME}/.streamlinkrc`
+  - Linux/BSD:
+    - `${XDG_CONFIG_HOME:-${HOME}/.config}/streamlink/plugins`
+    - `${HOME}/.streamlinkrc`
+
   Support for these old paths will be dropped in the future.  
   See the [CLI documentation](https://streamlink.github.io/cli.html) for all the details regarding these changes.
 - Implemented: `--logfile` CLI argument ([#3753](https://github.com/streamlink/streamlink/pull/3753))
```

**Comment by bastimeyer** [comment]
@gravyboat Are we good here? I'd like to get this merged and released as soon as possible, to fix the Youtube issues. If there's no response in the next couple of hours, I'm going to merge myself. :/

File: CHANGELOG.md (hunk 1 above)
Comment (gravyboat) [reply CHANGELOG.md]: __

**Comment by gravyboat** [comment]
@bastimeyer Yes we're good, sorry I was busy at work. Just threw that comment in this morning to try and make sure things were sorted in case it was merged during the day.

**Comment by gravyboat** [comment]
@bastimeyer Tagged and pushed.

**Comment by gravyboat** [comment]
@bastimeyer Just loaded it up on my Windows machine and things seem good. It didn't fall back to my streamlinkrc and instead loaded the default config which was expected but some people might have issues. Should we make an issue and pin it just in case?

**Comment by bastimeyer** [comment]
Yes, probably a good idea.
I was also thinking about this Windows installer config file change earlier, but a workaround that renames old config files is not really a good solution, so this is unfortunately a small breaking change.

**Comment by gravyboat** [comment]
People can always delete it if they want the old `streamlinkrc` to be picked up. I'll make a small issue noting they can either copy the config or remove the config file.

**Pull request merged**